score/autonomy.go

```diff
@@ -9,3 +11,33 @@ func CalculateIndividualAutonomyScore(individualMetric schema.IndividualMetric,
 
 	return scoreToday, ChangeRate(float64(scoreToday), float64(scoreYesterday))
 }
+
+func CalculatePOIAutonomyScore(resources []schema.POIResourceRating, neighbor schema.Metric) (float64, float64) {
+	sumOfScoreToday := float64(0)
+	sumOfScoreYesterday := float64(0)
+	sumOfRatingsToday := float64(0)
+	sumOfRatingsYesterday := float64(0)
+	scoreToday := float64(0)
+	scoreYesterday := float64(0)
+
+	for _, r := range resources {
+		if 0 == r.Ratings {
+			continue
+		}
+		sumOfScoreToday = sumOfScoreToday + r.Score*float64(r.Ratings)
```

Comment:
```suggestion
		sumOfScoreToday = sumOfScoreToday + r.Score*float64(r.Score)
```

`r.Ratings` actually means the number of people who've made their ratings?

Reply:
Yes. I think so.